layouts/_default/baseof.html

```diff
@@ -165,7 +165,7 @@ <h1 data-pagefind-meta="title">About{{ if (isset .Params "subtitle") }} - {{ .Pa
           {{ end }}
         </div>
       </div>
-      {{ partial "footer.html" . }}
+      {{ partialCached "footer.html" . }}
```

Comment:
I believe that this would break the `/search` page, as there is a line in `footer.html` that is contingent on the current page section.

Reply:
@dotnetCarpenter could you have a look whether the `search` page renders correctly?

Reply:
@dotnetCarpenter I had a look, and it does not look good:

![ca70cdd917e0fef93cec35d593cd9e12db7a5ab7](https://github.com/user-attachments/assets/c4d6afcd-429b-49f8-92c6-4cb530c36b5b)

In my hands, you need this to fix it:

```suggestion
      {{ if eq (.Scratch.Get "section") "search" }}
      {{ partial "footer.html" . }}
      {{ else }}
      {{ partialCached "footer.html" . }}
      {{ end }}
```

Reply:
@dotnetCarpenter see https://github.com/git/git-scm.com/pull/1955.

Reply:
@dscho I'll take a look :)